### ISSUE TITLE
Support for Trac ticket

### ISSUE THREAD

**Issue by Akasurde** [dmranck/ticketutil#4]
As $subj, can we have support for trac tickets as well.

**Comment by dmranck** [comment]
Yes, that's on our list of tools to support in the near future. Thanks for the feedback!

**Comment by Akasurde** [comment]
@dmranck Let me know if you need a hand in this.

**Comment by dmranck** [comment]
@Akasurde If you're interested in adding support for Trac, feel free! It's in our backlog but probably won't be worked on for another month or two.
